Gamma_Analysis.py

```diff
@@ -127,6 +127,10 @@ def isotope_concentration(isotope, reference, sample_activity,
         conversion = 1
     not_in_dirt = ['Cs134', 'Cs137', 'Co60', 'Pb210']
     ref_specific_activity = reference_activity[0] / reference.mass
+
+    if type(reference) is ref.base_reference_class():
+    elif type(reference) is ref.petri_reference_class():
```

Comment:
Two things about classes:
1. Standard python style is to use CamelCase for class names. So you could have `class ReferenceBase(object):` and `class PetriReference(ReferenceBase):` or so. You don't need the word "class" in the name because the CamelCase reminds you that it's a class.
2. When checking class, it's considered better practice to use `isinstance` instead of comparing `type`. If the petri class is a subclass of the main reference class, you do need to be careful since `isinstance(reference, ReferenceBase)` will be true for both a petri reference and a regular reference. So you can do `if isinstance(reference, PetriReference):` and then `else:`